src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
```

Comment:
It is recommended to use BEM naming convention for the class names. In this case, you can use 'todo-list' instead of 'TodoList'.